nixos/doc/manual/release-notes/rl-2405.section.md

```diff
@@ -394,6 +394,8 @@ The pre-existing [services.ankisyncd](#opt-services.ankisyncd.enable) has been m
 
   `redirectCode`.
 
+- `restic` module now has an option for inhibiting system sleep while backups are running, defaulting to off (not inhibiting sleep), available as [`services.restic.backups.<name>.inhibitsSleep`](#opt-services.restic.backups._name_.inhibitsSleep).
```

Comment:
this documents the option more clearly than the option's actual documentation, although `false = don't inhibit` should probably be obvious from the option name, so i its probably fine as-is.

Reply:
What do you think about the updated option description?